src/main/java/io/fabric8/maven/docker/util/AuthConfigFactory.java

```diff
@@ -207,10 +207,12 @@ private AuthConfig createStandardAuthConfig(boolean isPush, Map authConfigMap, S
             }
 
             // Check for openshift authentication either from the plugin config or from system props
-            ret = getAuthConfigFromOpenShiftConfig(lookupMode, authConfigMap);
-            if (ret != null) {
-                log.debug("AuthConfig: OpenShift credentials");
-                return ret;
+            if (lookupMode != LookupMode.REGISTRY) {
```

Comment:
I would skip that as it doesn't harm to use the openshift lookup with a registry. prefix, too. Its probably more confusing to have this special case here.

Reply:
The reason why I excluded `LookupMode.REGISTRY` is:
When the config contains `useOpenShiftAuth=true` and the system properties contain `docker.useOpenShiftAuth=false` `createStandardAuthConfig` returns a (valid) configuration (see `io.fabric8.maven.docker.util.AuthConfigFactoryTest#testOpenShiftConfigFromSystemPropsNegative` which fails if I remove the if)